pages/edge/quickstart/js-wintercg.mdx

```diff
@@ -0,0 +1,98 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Quickstart guide for deploying a JS Service Worker
+
+In this guide, you'll learn the process of deploying a javascript service worker on Wasmer Edge.
+
+## Deploying a JavaScript Service Worker
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Step 3.1: Initialize the JavaScript Service Worker Starter template
+
+```shell copy
+$ wasmer app create
+App type:
+  Static website
+  HTTP server
+  Browser shell
+> JS Worker (experimental)
+  Python Worker
```

Comment:
This should be:

```
  Python Webserver
```

Reply:
Nothing can be done till next wasmer release